packages/lib/serverConfig.ts

```diff
@@ -19,10 +19,18 @@ function detectTransport(): SendmailTransport.Options | SMTPConnection.Options |
       },
       secure: port === 465,
       tls: {
-        rejectUnauthorized: isENVDev ? false : true,
+        rejectUnauthorized: !isENVDev,
       },
     };
 
+    if (isENVDev) {
+      transport = {
+        port: process.env.EMAIL_SERVER_PORT,
```

Comment:
This does the same as the port parameter, what is this fixing?

Reply:
Mailhog wasn't working for me locally when I was using the rest of the parameters in transport, like host, auth, secure, etc. After searching a lot, I found that just mentioning the port is enough for mailhog to work. I will revert these changes if this issue is only specific to my system. Can you pls confirm this? @emrysal